application/src/main/java/org/togetherjava/tjbot/logging/discord/DiscordLogForwarder.java

```diff
@@ -193,6 +199,14 @@ private static String describeLogEvent(LogEvent event) {
             return logMessage + "\n" + exceptionWriter.toString().replace("\t", "> ");
         }
 
+        private static @Nullable String linkToSource(String source, String sourceCodeBaseUrl) {
```

Comment:
return Optional<String> instead. on the call-site, u can then use `.orElse(null)`